v4l2/v4l2_codecbase.cpp

```diff
@@ -279,12 +281,13 @@ const char* V4l2CodecBase::IoctlCommandString(int command)
 }
 #endif
 
-int32_t V4l2CodecBase::ioctl(int command, void* arg)
+int32_t V4l2CodecBase::ioctl(uint64_t command, void* arg)
```

Comment:
Since it only cause problem when you use printf. Could we make some workaround in printf format? We'd better not change v4l2_device.h,since we chrome authors is owner for this file